src/test/java/api/support/fixtures/FeeFineOwnerFixture.java

```diff
@@ -13,7 +13,7 @@
   private final ServicePointsFixture servicePointsFixture;
 
   public FeeFineOwnerFixture() {
-    super(forFeeFineOwners(), json -> json.getJsonArray("servicePointOwner").toString());
+    super(forFeeFineOwners(), json -> json.getString("owner"));
```

Comment:
Has the structure of a fee fine owner changed, or was this using the incorrect definition before?

Reply:
We had incorrect definition before. There is unique key for the owner property not for the servicePointOwner